beetsplug/mbsubmit.py

```diff
@@ -56,7 +60,44 @@ def __init__(self):
 
     def before_choose_candidate_event(self, session, task):
         if task.rec <= self.threshold:
-            return [PromptChoice("p", "Print tracks", self.print_tracks)]
+            return [
+                PromptChoice("p", "Print tracks", self.print_tracks),
+                PromptChoice(
+                    "f",
+                    'print tracks into a "beets-unmatched" File',
+                    self.markunmatched,
+                ),
+                PromptChoice(
+                    "o", "Open Picard with the relevent files", self.picard
+                ),
+            ]
+
+    def markunmatched(self, session, task):
+        fmt = self.config["format"].as_str()
+        for item in task.items:
+            unmatched_path = (
+                Path(item.path.decode("utf-8")).parent / "beets-unmatched"
+            )
+            with open(unmatched_path, "a") as p:
+                p.write(format(item, fmt) + "\n")
+
+    def picard(self, session, task):
+        picard_path = which(self.config["picard-path"].as_str())
+        if picard_path is None:
+            if picard_path.startswith("/"):
```

Comment:
This second if-condition will never return True if picard_path was None already. Maybe u wanted to use `if picard_path is not None`

Reply:
> This second if-condition will never return True if picard_path was None already. Maybe u wanted to use `if picard_path is not None`

Thanks for pointing that out, I changed the logic there to what I meant.